Failure to compile on CentOS8

I have all GCC dependencies installed and have tried compiling other software with no issues.

output of make: 
[me@host:avra]$ make
make -C src -f makefiles/Makefile.linux
make[1]: Entering directory '/home/acopeland/avra/src'
gcc -static -o avra avra.o device.o parser.o expr.o mnemonic.o directiv.o macro.o file.o map.o coff.o args.o stdextra.o -s
/usr/bin/ld: cannot find -lc
collect2: error: ld returned 1 exit status
make[1]: *** [makefiles/Makefile.linux:22: avra] Error 1
make[1]: Leaving directory '/home/acopeland/avra/src'
make: *** [Makefile:23: all] Error 2

--- a/src/makefiles/Makefile.linux
+++ b/src/makefiles/Makefile.linux
@@ -1,7 +1,7 @@
-CC = gcc
-LD = gcc -static
-CFLAGS = -Wall -O3 $(CDEFS)
-LDFLAGS = -s
+CC ?= gcc
+CFLAGS ?= -Wall -O3
+CFLAGS += $(CDEFS)
+LDFLAGS ?= -s
 
 SOURCES = avra.c \
 	device.c \
@@ -19,7 +19,7 @@ SOURCES = avra.c \
 OBJECTS = $(SOURCES:.c=.o)
 
 avra: $(OBJECTS)
-	$(LD) -o $@ $(OBJECTS) $(LDFLAGS)
+	$(CC) -o $@ $(OBJECTS) $(LDFLAGS)
 
 clean:
 	rm -f avra *.o *.p *~

